packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -283,6 +284,9 @@ class TransactionService {
           if (name === NAME_UNKNOWN && pageLoadTransactionName) {
             tr.name = pageLoadTransactionName
           }
+
+          tr.parentId = tr.options.pageLoadParentId
```

Comment:
Lets move this to the capture Navigation where we set the traceid and span id details - https://github.com/elastic/apm-agent-rum-js/blob/6464a1d312d5ff1e2591ef3b6979ce9171916cf8/packages/rum-core/src/performance-monitoring/navigation/capture-navigation.js#L66. 

Could you also add a comment that this parentID logic helps with crosslinking Synthetics tests etc.

Reply:
done!